onnxruntime/contrib_ops/cpu/attnlstm/attention_wrapper.cc

```diff
@@ -51,7 +51,7 @@ void AttentionWrapper<T>::ProcessOutput(const gsl::span<const T>& rnn_cell_outpu
   // Get the context which is calculated within attention mechanism.
   attention_mechanism_.Compute(rnn_cell_output, prev_alignments_, attn_context_, alignments_);
   if (attention_mechanism_.NeedPrevAlignment()) {
-    std::copy(alignments_.cbegin(), alignments_.cend(), prev_alignments_.begin());
+    std::copy(alignments_.begin(), alignments_.end(), prev_alignments_.begin());
```

Comment:
MS GSL doesn't have gsl::span::cbegin and gsl::span::cend.

note that they are also not present in std::span.